doc/neps/nep-0052-python-api-cleanup.rst

```diff
@@ -122,53 +147,77 @@ probably. Here is a representative set of examples:
   and other issues for ``maximum_sctype`` and related functions).
 - Business day functionality can likely be removed (unclear if it needs
   splitting out like was done for ``np.financial``).
-- The ``np.compat`` namespace can be removed.
-- There are lots of one-off functions that can be deprecated and removed, such as
-  ``real_if_close`` (see `gh-11375 <https://github.com/numpy/numpy/issues/11375>`__).
-  These can be identified via triaging the issue tracker and/or via going
-  through the main namespace manually.
 
-There are new namespaces for warnings/exceptions (``np.exceptions``) and for dtype-related
-functionality (``np.types``). NumPy 2.0 is a good opportunity to move things
-there from the main namespace.
+S: I think `np.financial` maintainer was interested in adopting?
+
+- The ``np.compat`` namespace, used during the Python 2 to 3 transition, will be removed.
+- Functions that are narrow in scope, with very few public use-cases,
+  will be removed.  See, e.g.
+  ``real_if_close`` (`gh-11375 <https://github.com/numpy/numpy/issues/11375>`__).
+  These will have to be identified manually and by issue triage.
+
+New namespaces are introduced for warnings/exceptions (``np.exceptions``) and for dtype-related
+functionality (``np.types``). NumPy 2.0 is a good opportunity to
+popular these submodules from the main namespace.
 
-Functionality that gets a fair amount of usage but has a preferred alternative
+.. S: Has the ``np.types`` name been fixed? Wonder if we're going to
```

Comment:
Sebastian's PR is still open, so it can still be changed. Not sure I have a better suggestion though, it seems okay to me.